dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/statelisteners/ElasticListener.java

```diff
@@ -257,9 +260,6 @@ public void afterBulk(long executionId, BulkRequest request,
                     LOGGER.error("Could not submit " + index + " index to elastic search in time");
                     throw new CustomWebApplicationException("Could not submit " + index + " index to elastic search in time", HttpStatus.SC_INTERNAL_SERVER_ERROR);
                 }
-                if (!afterBulkFailures.isEmpty()) {
```

Comment:
A reindex operation now reindexes in [batches](https://github.com/dockstore/dockstore/blob/c9106e48c6d75cb1bacfd9737910fde334a37bf8/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsApiExtendedServiceImpl.java#L190), so while the batch will complete, the `throw` would now stop the rest of the batches from getting indexed.

I hadn't noticed the LGTM alert, thanks -- I guess because all tests are failing due to detect_migrations :(, so I'll look at both of those.

On the type of error, this was the one I was able to reproduce locally with a copy of the dev data. After the fix, locally, the total number of indexed entries went up significantly, as the batch containing the problematic workflow was in one of the middle batches.